### PR TITLE
Add ABNF to express service type field

### PR DESCRIPTION
**Pull request by EzequielPostan**
Fix ATL-4660

### CODE REVIEW

File: w3c-spec/PRISM-method.md [hunk 1]
```diff
@@ -327,6 +327,11 @@ Below we see the rules to construct a well-formed `CreateDIDOperation`, these ru
       - The `type` value MUST not start nor end with whitespaces, and MUST have at least a non whitespace character. 
       - The `type` value MUST not exceed `MAX_TYPE_SIZE` characters in length.
       - The strings constructing the value in `type` SHOULD be registered in the DID Specification Registries [DID-SPEC-REGISTRIES](https://www.w3.org/TR/did-spec-registries/)
+      - To be more precise, the `type` value MUST conform the following ABNF rule
+            
+              type-value  = type-string / "[" *(type-string ",") type-string "]"
```
Comment (patlo-iog) [inline w3c-spec/PRISM-method.md:332]: Does this part `"[" *(type-string ",") type-string "]"`  contradicts with the statement above?

> The type field MUST be a string or a non-empty JSON array of strings

This ABNF allows a value of
```scala
val typeValue: String = "[apple,banana]"
```

while a valid JSON array of strings looks like this
```scala
val typeValue: String = "[\"apple\", \"banana\"]"
```

This should align with the statement above?
```abnf
type-value = type-string / "[" *(quoted-type-string ", ") quoted-type-string "]"
quoted-type-string =  DQUOTE type-string DQUOTE
```
and optionally add/omit the `*SP` between items in an array to refine whitespace sensitivity.

Comment (EzequielPostan) [reply w3c-spec/PRISM-method.md:332]: thank you @patlo-iog , I always forget the string quotes
W.r.t. optional `SP` characters between array elements, I think it is better to not allow them, DID resolvers can add the spaces at their end for readabillity 
do you have a reason why allowing them would be desirable?

Comment (patlo-iog) [reply w3c-spec/PRISM-method.md:332]: I also prefer not to allow them as well. So it will be a more strict JSON array.

**Review by shotexa — APPROVED** [review]
I only had one clarifying question, everything else looks good

File: w3c-spec/PRISM-method.md (hunk 1 above)
Comment (EzequielPostan) [reply w3c-spec/PRISM-method.md:332]: today, we allow it, but I am not aware of any service using that at the time of this writing 
Should we remove them?

**Pull request merged**